File: test-base/src/main/java/kin/ecosystem/test/base/BaseTestClass.java
package kin.ecosystem.test.base;

import com.kin.ecosystem.core.bi.EventsStore;
import org.junit.Before;


public class BaseTestClass {

	protected UserModifierFake userModifierFake = new UserModifierFake();
	protected CommonModifierFake commonModifierFake = new CommonModifierFake();
	protected ClientModifierFake clientModifierFake = new ClientModifierFake();

	@Before
	public void setUp() throws Exception {
		setUpEventsCommonData();
	}

	private void setUpEventsCommonData() {
		EventsStore.init(userModifierFake, commonModifierFake, clientModifierFake);
	}
}

File: test-base/src/main/java/kin/ecosystem/test/base/BaseTestClass.kt
package kin.ecosystem.test.base

import com.kin.ecosystem.core.bi.EventsStore
import org.junit.Before


open class BaseTestClass {

    private val userModifierFake = UserModifierFake()
    private val commonModifierFake = CommonModifierFake()
    private val clientModifierFake = ClientModifierFake()

    @Before
    @Throws(Exception::class)
    open fun setUp() {
        setUpEventsCommonData()
    }

    private fun setUpEventsCommonData() {
        EventsStore.init(userModifierFake, commonModifierFake, clientModifierFake)
    }
}
